meeting/meeting_july_6_2021.md

```diff
@@ -0,0 +1,85 @@
+# July 6th (7 PM CST)/6th (9 AM JST) Meeting 2021
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Tibo)**
+* **#Query(Lloyd)**
+* **#Query(Chris)**
+* **#Query(Antoine)**
+* **#Query(Jesse)**
+* **#Query(Matt)**
+* **#Query(Ben)**
+* **#Query(Ivan!)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+* rust-dlc
+* atomic finance
+* others?
+
+## Specification Writing
+
+**#Status_Update_Interrupt**
+
+* Multi-oracle support merged!
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/128
+* Hyperbola curve piece support merged!
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/144
+* If anyone is interested it would be nice to document loss-of-fund vectors
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/132
```

Comment:
Something I thought of and that could potentially be a vector for loss of fund is being robust about random failures, would like to discuss if people have considered that.